bin/did-whisper

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env node
+const yargs = require('yargs');
+const getStdin = require('get-stdin');
+
+const whisper = require('../lib/index');
+
+const DEFAULT_URL = 'http://localhost:5000';
```

Comment:
I'm thinking that the default should be https://whisper.demo.veres.one

Which would allow someone to most easily try this thing out without being concerned at all about specifying a storage location.  Whereas now, the default expects one to be running the whisper-server locally.

It does seem like a good idea to guide devs to the proper settings for dev mode as well.  So I don't know, maybe a --dev option that sets the default port to what it is now unless it's overridden by the -s option?